### ISSUE TITLE
stopped working in recent versions of firefox

### ISSUE THREAD

**Issue by eigerzoom** [Infocatcher/Right_Links#28]
I must downgrade to firefox verson 44.0.2 to make rightlinks work again
i've tried changing many custom settings in extensions.rightlinks in about:config
but nothing has made it function properly again.

**Comment by Infocatcher** [comment]
I think, this is due to enabled multi-process mode:
https://github.com/Infocatcher/Right_Links/issues/17

**Comment by eigerzoom** [comment]
can that be disabled?

**Comment by eigerzoom** [comment]
also to note, it seems to work on bookmarks, history, just not links in web pages, but i did notice on some links it does work, like if open about:addons and click on your infocatcher add-on, I can rightlink the infocatcher link there for some reason

**Comment by eigerzoom** [comment]
yeah i just fixed it disabling multi-process mode (browser.tabs.remote)
about:config
browser.tabs.remote
made all sensible entries false instead of true

**Comment by mustaqimM** [comment]
@Infocatcher It seems to have stopped again on the latest Nightly.
